docs/tutorials/pre_executed/using_rubin_data.ipynb

```diff
@@ -0,0 +1,404 @@
+{
```

Comment:
<p>more explanation of nested columns - link to nested documentation or similar. This might be first time an user sees this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>